src/scenes/MerchStorePage/getObjectName.ts

```diff
@@ -0,0 +1,28 @@
+import { graphql, readInlineData } from 'relay-runtime';
+
+import { getObjectNameFragment$key } from '~/generated/getObjectNameFragment.graphql';
+
+const MERCHS_NAMING = {
+  TShirt: '(OBJECT 001) Shirt',
+  Hat: '(OBJECT 002) hat',
+  Card: '(OBJECT 003) card ',
```

Comment:
can we keep the capitalization consistent? `hat -> Hat`, `card -> Card`

Reply:
Sure!